djangocms_versioning/datastructures.py

```diff
@@ -11,6 +14,25 @@ def __init__(self, content_model, grouper_field_name):
     def _get_grouper_field(self):
         return self.content_model._meta.get_field(self.grouper_field_name)
 
+    @cached_property
+    def version_proxy(self):
```

Comment:
I would change this to `version_model_proxy` as it's actually very important to the understanding of this code that it returns a model class. And it would be good to add a docstring as well as it's really not a straightforward property! :)